add-on/manifest.json

```diff
@@ -60,6 +60,7 @@
         {
             "matches": [
               "https://play.google.com/music/listen*",
+              "https://podcasts.google.com/*",
```

Comment:
You also need to add this bit higher, to "permissions" – without this addon won't be able to inject those scripts.